test/unit/plugin/v2/installer_test.rb

```diff
@@ -51,18 +58,19 @@ def teardown
       Dir.glob(File.join(@config_dir_path, 'empty', '*')).each do |path|
         next if path.end_with? '.gitkeep'
         FileUtils.rm_rf(path)
-      end
+      end if @config_dir_path
     end
 
     # Clean up any activated gems
     Gem.loaded_specs.delete('inspec-test-fixture')
+    Gem.loaded_specs.delete('ordinal_array')
```

Comment:
I think I know what this fixes, but just for completeness, could you comment explaining what this is for?

Reply:
What are you looking for here beyond the comment above it? That some tests activate different dependencies and will wind up causing test ordering issues?